src/main/js/dynamic/loaded/users/PermissionTable.js

```diff
@@ -4,6 +4,7 @@ import TableDisplay from 'field/TableDisplay';
 import DynamicSelectInput from 'field/input/DynamicSelect';
 import CheckboxInput from 'field/input/CheckboxInput';
 import { CONTEXT_TYPE } from 'util/descriptorUtilities';
+import StatusMessage from '../../../field/StatusMessage';
```

Comment:
This import should be absolute not relative

Reply:
Do our lint settings cover this?

Reply:
The linting thinks that the absolute paths are wrong because it cannot resolve the path from the current location where the file is located.  Currently it does not catch this.